ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/member/entities/Member.java

```diff
@@ -10,17 +10,19 @@
 
 public class Member {
 
-	private final String collectionName;
 	private final Model memberModel;
+	private final String collectionName;
+	private final Long index;
 	private final String memberId;
 	private final String versionOf;
 	private final LocalDateTime timestamp;
 	private final List<String> treeNodeReferences;
 
-	public Member(String collectionName, String memberId, String versionOf, LocalDateTime timestamp,
+	public Member(String memberId, String collectionName, Long index, String versionOf, LocalDateTime timestamp,
 			final Model memberModel, List<String> treeNodeReferences) {
 		this.collectionName = collectionName;
 		this.memberId = memberId;
+		this.index = index;
```

Comment:
I don't know if 'index' is the best name for this. I associate this more with sequences than indices. Wouldn't sequenceNr  be more explicite?

Reply:
Updated to `sequenceNr`